metis-indexing/metis-indexing-service/src/main/java/eu/europeana/indexing/service/dao/FullBeanDao.java

```diff
@@ -0,0 +1,126 @@
+/*

```

Comment:
Ok so I see that in the FullBeanDao the connection to the database is initialized.
The connection to the database should be initialized in a different class, maybe IndexingMongoServer. (Similar way to MorphiaDatastoreProvider, search on the project is in the metis-core)

And the FullBeanDao which is a DAO should be provided with a connection and that connection should be used for any DAO operation(DatasetDao).